v2/pkg/genruntime/configmaps/validation.go

```diff
@@ -21,6 +22,11 @@ type keyPair struct {
 // ValidateDestinations checks that no two destinations are writing to the same configmap/key, as that could cause
 // those values to overwrite one another.
 func ValidateDestinations(destinations []*genruntime.ConfigMapDestination) (admission.Warnings, error) {
+	return ValidateDestinationsExt(destinations, nil)
+}
+
+// TODO: ValidateDestinationsExt will replace ValidateDestinations in a future PR.
+func ValidateDestinationsExt(destinations []*genruntime.ConfigMapDestination, destinationExpressions []*core.DestinationExpression) (admission.Warnings, error) {
```

Comment:
Consider formatting these parameters on different lines to make the declaration easier to read.

```suggestion
func ValidateDestinationsExt(
        destinations []*genruntime.ConfigMapDestination, 
        destinationExpressions []*core.DestinationExpression,
) (admission.Warnings, error) {
```